### PR TITLE
Blog post to introduce the Alerts Configuration Manager

### PR DESCRIPTION
**Pull request by sashwathn**

### CODE REVIEW

File: blog/2023-12-21-alert-configuration-manager/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+description: |
+  Discover the power of Netdata's new Alerts Configuration Manager, a user-friendly UI wizard for creating and managing alerts directly from your Netdata Dashboard.
+authors: satya
+tags: [netdata, alerts-configuration-manager, UI-dashboard, alert-management]
+keywords: [netdata, alerts configuration, UI wizard, dashboard, alert management]
+image: ./img/alert-config-manager.png
+---
+
+
+slug: netdata-cost-transparency-unveiling-the-true-cost-of-monitoring
```
Comment (hugovalente-pm) [inline blog/2023-12-21-alert-configuration-manager/index.md]: all this and below seems to be a duplicate from previous blogpost

File: blog/2023-12-21-alert-configuration-manager/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (hugovalente-pm) [inline blog/2023-12-21-alert-configuration-manager/index.md]: there is something acting strange with the slug/metadata section and I think it is this empty line on top
see an example of another blogpost
image

Comment (sashwathn) [reply blog/2023-12-21-alert-configuration-manager/index.md]: I fixed it the `---` were not on the first line.. :D

**Comment by hugovalente-pm** [comment]
images overall seem large but looking to other blogpost when they go-live they seem to be on a proper size
let's verify once it is out

File: blog/2023-12-21-alert-configuration-manager/index.md (hunk 1 above)
Comment (sashwathn) [reply blog/2023-12-21-alert-configuration-manager/index.md]: This was me checking what was wrong.. ;)

**Comment by sashwathn** [comment]
> images overall seem large but looking to other blogpost when they go-live they seem to be on a proper size let's verify once it is out

There is this one image of copying the alert configuration which is large.. I can edit the image but I wanted to show the `Configuration Copied` message.

**Review by hugovalente-pm — APPROVED** [review]

**Pull request merged**